projects/1.20/assets/domum-ornamentum/domum_ornamentum/lang/zh_cn.json

```
@@ -0,0 +1,210 @@
+{
+  "__comment": "以上仅限数据生成条目",
+  "block.domum_ornamentum.architectscutter": "建筑师切割机",
+  "block.domum_ornamentum.beige_bricks": "米黄色砖块",
+  "block.domum_ornamentum.beige_stone_bricks": "米黄色石砖",
+  "block.domum_ornamentum.black_floating_carpet": "黑色浮空地毯",
+  "block.domum_ornamentum.blockbarreldeco_onside": "横放木桶",
+  "block.domum_ornamentum.blockbarreldeco_standing": "竖放木桶",
+  "block.domum_ornamentum.blue_floating_carpet": "蓝色浮空地毯",
+  "block.domum_ornamentum.brown_bricks": "棕色砖块",
+  "block.domum_ornamentum.brown_floating_carpet": "棕色浮空地毯",
+  "block.domum_ornamentum.brown_stone_bricks": "棕色石砖",
+  "block.domum_ornamentum.cream_bricks": "奶油色砖块",
+  "block.domum_ornamentum.cream_stone_bricks": "奶油色石砖",
+  "block.domum_ornamentum.roan_bricks": "斑驳砖块",
+  "block.domum_ornamentum.roan_stone_bricks": "斑驳石砖",
+  "block.domum_ornamentum.cyan_floating_carpet": "青色浮空地毯",
+  "block.domum_ornamentum.gray_floating_carpet": "灰色浮空地毯",
+  "block.domum_ornamentum.green_floating_carpet": "绿色浮空地毯",
+  "block.domum_ornamentum.light_blue_floating_carpet": "淡蓝色浮空地毯",
+  "block.domum_ornamentum.light_gray_floating_carpet": "淡灰色浮空地毯",
+  "block.domum_ornamentum.lime_floating_carpet": "黄绿色浮空地毯",
+  "block.domum_ornamentum.magenta_floating_carpet": "品红色浮空地毯",
+  "block.domum_ornamentum.orange_floating_carpet": "橙色浮空地毯",
+  "block.domum_ornamentum.pink_floating_carpet": "粉红色浮空地毯",
+  "block.domum_ornamentum.purple_floating_carpet": "紫色浮空地毯",
+  "block.domum_ornamentum.red_floating_carpet": "红色浮空地毯",
+  "block.domum_ornamentum.sand_bricks": "沙砖",
+  "block.domum_ornamentum.sand_stone_bricks": "沙石砖",
+  "block.domum_ornamentum.white_floating_carpet": "白色浮空地毯",
+  "block.domum_ornamentum.yellow_floating_carpet": "黄色浮空地毯",
+  "cuttergroup.domum_ornamentum.avanilla": "方块结构",
+  "cuttergroup.domum_ornamentum.btimberframe": "框架方块",
+  "cuttergroup.domum_ornamentum.cshingle": "瓦片",
+  "cuttergroup.domum_ornamentum.ddoor": "门",
+  "cuttergroup.domum_ornamentum.etrapdoor": "活板门",
+  "cuttergroup.domum_ornamentum.fpanel": "面板",
+  "cuttergroup.domum_ornamentum.gpillar": "柱子",
+  "cuttergroup.domum_ornamentum.hpaperwall": "薄墙",
+  "cuttergroup.domum_ornamentum.ilight": "灯",
+  "cuttergroup.domum_ornamentum.jbrick": "砖块",
+  "cuttergroup.domum_ornamentum.kpost": "桩子",
+  "domum_ornamentum.allbrick.column.format": "主要材料: %s",
+  "domum_ornamentum.architectscutter": "建筑师切割机",
+  "domum_ornamentum.block.format": "材料：%s",
+  "domum_ornamentum.blockpillar.name.format": "圆形%s柱子",
+  "domum_ornamentum.blockypillar.name.format": "体素%s柱子",
+  "domum_ornamentum.dark_brick.name.format": "暗纹%s砖块",
+  "domum_ornamentum.dark_brick_stair.name.format": "暗纹%s砖楼梯",
+  "domum_ornamentum.desc.center": "中心%s",
+  "domum_ornamentum.desc.frame": "边框%s",
+  "domum_ornamentum.desc.main": "主要%s",
+  "domum_ornamentum.desc.material": "材料: %s",

```

```suggestion
  "domum_ornamentum.desc.material": "材料：%s",
```
其余引号后空格也请删除。但为保持格式所使用的`  -  `中的空格无需删除。